### PR TITLE
GRAPE-19393

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

**Review by pseudonymcp — CHANGES_REQUESTED** [review]
Label and help text should be vertically aligned on the left side (currently the the help text starts shifted a bit to the right compared to the label). Appears in Docusaurus -> improve styling to not be influenced by outside styles.

**Review by pseudonymcp — APPROVED** [review]
Looks good from the visual side 👍

File: web/src/components/checkbox/index.tsx [hunk 1]
```diff
@@ -50,19 +50,23 @@ export const Checkbox: React.FC<CheckboxProps> = props => {
     isChecked: state.isSelected,
   })
   const hasLabel = isLabelCheckbox(props)
+  const labelId = genUid()
```
Comment (sk7) [inline web/src/components/checkbox/index.tsx:53]: seems like the `labelId` gets re-generated every time I click the checkbox, why?

Comment (AleshaOleg) [reply web/src/components/checkbox/index.tsx:53]: Right, I need to use memoization. Thanks!

**Review by sk7 — APPROVED** [review]

**Pull request merged**